lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -17,7 +17,7 @@ import 'pointer_binding.dart';
 import 'safe_browser_api.dart';
 import 'semantics.dart';
 import 'text_editing/text_editing.dart';
-import 'util.dart';
+import 'view_embedder/application_dom/application_dom.dart';
```

Comment:
Please, **do** look at this file, even if github doesn't want to show it to you by default!